test/test_torch.py

```diff
@@ -1126,18 +1126,25 @@ def test_pdist_empty(self):
             self.assertEqual(torch.zeros(3, device=device), torch.pdist(x))
 
     def test_pdist_norm(self):
+        def test_pdist_single(shape, device, p, trans):
```

Comment:
as above, it's probably good to also grid over the relevant dtypes

Reply:
done.